### PR TITLE
JIRA[ELPPCKVM-971]:[postcopy]Add polarion case rhel7_49062 and rhel7_85702

### PR DESCRIPTION
**Pull request by wangxianxian**
Signed-off-by: Xianwang <xianwang@redhat.com>

### CODE REVIEW

**Comment by wangxianxian** [comment]
<dgilbert> xianwang: I can see why you're confused; but that output is OK;  'remaining' is just the amount of data we know we have to transmit, I think you probably caught it while it was scanning for new dirty pages, so it didn't yet know how many more to transmit
<xianwang> dgilbert, ok, thanks, but I want to know which one is more precise for checking whether migration is completed, maybe "status": "completed" is more reliable than "remaining"?
<dgilbert> xianwang: Yes, always status
<xianwang> dgilbert, ok, got it, thanks
<dgilbert> xianwang: the way 'remaining' works is that we update it, send all the 'remaining' and then we update remaining again
<dgilbert> xianwang: So it always goes down and then back up, and then down again....
<xianwang> dgilbert, oh, yes
<xianwang> but once "status" is "completed", it could indicate migration is finished 
<dgilbert> right
I hit once about checking migration status, that "status": "active", but "remaining": 0, so, according to Dave,  I think it is needed to update line28 of utils_migration.py

**Comment by wangxianxian** [comment]
@huangyum 
yumei,  please help to review this patch, although I have tried it pass both on x86_64 and ppc, thanks

**Comment by huangyum** [comment]
# git am 218.patch
Applying: JIRA[ELPPCKVM-971]:[postcopy]Add polarion case rhel7_49062 and rhel7_85702
/home/git/mouse/.git/rebase-apply/patch:91: new blank line at EOF.
+
/home/git/mouse/.git/rebase-apply/patch:224: new blank line at EOF.
+
/home/git/mouse/.git/rebase-apply/patch:349: new blank line at EOF.
+
/home/git/mouse/.git/rebase-apply/patch:472: new blank line at EOF.
+
/home/git/mouse/.git/rebase-apply/patch:712: new blank line at EOF.
+
warning: 5 lines add whitespace errors.

File: virtkvmqe/migration_test_plan/postcopy/test_requirement/x86_64/rhel7_49060_win.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ qmp:
+  - 'tcp:0:{qmp_port},server,nowait'
+ serial:
+  - 'tcp:0:{serial_port},server,nowait'
+ monitor:
+  - 'stdio'
+
+# Set the test case info linked to this test requirement.
+test_cases:
+ rhel7_10022_win:
```
[the remark this answers is not in the document]
Comment (wangxianxian) [reply virtkvmqe/migration_test_plan/postcopy/test_requirement/x86_64/rhel7_49060_win.yaml]: yes, these two cases are only for rhel guest, I just add postcopy yaml for  windows

**Comment by huangyum** [comment]
==>TOTAL : 2
==>PASS : 2 
   1: RHEL7-49062-[postcopy] Live migration with postcopy mode-two scenario(simpler stress/high stress) (13 min 41 sec)
   2: RHEL7-85702-[postcopy] Post-copy migration with XBZRLE compression	 (2 min 40 sec)
==>ERROR : 0

**Comment by wangxianxian** [comment]
@YongxueHong 
yongxue, please help to review this patch if you are convenient, thanks

**Pull request merged**